website/common/locales/en/groups.json

```diff
@@ -111,6 +111,20 @@
   "sortHabitrpgLastLoggedIn": "Sort by last time user logged in",
   "ascendingSort": "Sort Ascending",
   "descendingSort": "Sort Descending",
+
+  "sortClass": "Class",
```

Comment:
for class and background we already have these strings as `background` and `class`

Reply:
I didn't know if there were any rules on how or where to create these. I did find the other string, but I created these for modularity and context. But if reusablity is preferable then I can adjust for those accordingly. Let me know what you decide and I can modify the code accordingly.

Reply:
we tend to create new strings when the context is different but in this case class and background are just in a list not in a more complex phrase so I'd say it's better not to duplicate them

Reply:
@paglias, thanks for the heads up. I have removed the extra strings.